test/scripts/deposit.js

```diff
@@ -0,0 +1,74 @@
+const Contract = require("@truffle/contract")
+
+const BatchExchange = Contract(require("@gnosis.pm/dex-contracts/build/contracts/BatchExchange"))
+const GnosisSafe = artifacts.require("GnosisSafe")
+const ProxyFactory = artifacts.require("GnosisSafeProxyFactory")
+
+const { addCustomMintableTokenToExchange, deploySafe } = require("../test_utils")
+const { deployFleetOfSafes, buildDepositFromList } = require("../../scripts/utils/trading_strategy_helpers")(web3, artifacts)
+const { waitForNSeconds, execTransaction } = require("../../scripts/utils/internals")(web3, artifacts)
+
+contract("Deposit scripts", function (accounts) {
+  let gnosisSafeMasterCopy
+  let proxyFactory
+  let exchange
+  const safeOwner = accounts[0]
+  beforeEach(async function () {
+    gnosisSafeMasterCopy = await GnosisSafe.new()
+    proxyFactory = await ProxyFactory.new()
+
+    BatchExchange.setProvider(web3.currentProvider)
+    exchange = await BatchExchange.deployed()
+  })
+
+  const setup = async function (numberOfBrackets, amounts) {
+    const masterSafe = await GnosisSafe.at(await deploySafe(gnosisSafeMasterCopy, proxyFactory, [safeOwner], 1))
+    const bracketAddresses = await deployFleetOfSafes(masterSafe.address, numberOfBrackets)
+    const tokenInfo = []
+    const deposits = []
+    for (const { tokenData = {}, amount } of amounts) {
+      const symbol = tokenData.symbol || "TEST"
+      const decimals = tokenData.decimals || 18
```

Comment:
We only accept a deposit file, so I'd say that at least the token decimals should not be optional and the test should fail if no decimals are provided.

Reply:
Technically, we only use WEI here anyWEI so decimals don't matter at all. I would like to leave it for now, because I plan to use decimals in a future change to deposits.